converter/har/converter.go

```diff
@@ -136,6 +153,15 @@ func Convert(h HAR, includeCodeCheck bool, returnOnFailedCheck bool, batchTime u
 						}
 					}
 				}
+
+				if e.Response.Headers != nil {
+					for _, header := range e.Response.Headers {
+						if header.Name == "Location" {
+							fmt.Fprintf(w, "\t\tredirectUrl = res.headers.Location;\n")
+							recordedRedirectUrl = header.Value
```

Comment:
We could add a `break` here to stop the for-loop early.

Reply:
sure, but why add the code complexity? this isnt "runtime" code so a few extra loops wont hurt anyone.. also, maybe we'll want to act on additional headers in the future (maybe unlikely but who knows...)

Reply:
Sure, but that's in a possible future :) It's not a big thing at all, just something I noted as a possible improvement for the present, remove unnecessary cpu cycles if possible.

Reply:
IMO its premature optimization, but I'll add it :)